src/app/shared/utils/consts.ts

```diff
@@ -50,6 +50,7 @@ export const Consts = {
         'fusionstorage-object': "FusionStorage Object",
         'ceph-s3': "Ceph S3",
         'gcp-s3': "GCP Storage",
-        'ibm-cos': "IBM COS"
+        'ibm-cos': "IBM COS",
+        'yig': "YIG"
```

Comment:
Have you uploaded a logo for the YIG backend?

Reply:
The logo of YIG has not been confirmed yet, and it will be uploaded after confirmation later

Reply:
Ok. Create an issue for this so that we can track it later.